pennylane/plugins/expt_tensornet.py

```diff
@@ -239,10 +237,22 @@ def ev(self, obs_nodes, wires):
         if np.abs(expval.imag) > tolerance:
             warnings.warn(
                 "Nonvanishing imaginary part {} in expectation value.".format(expval.imag),
-                RuntimeWarning,
+                RuntimeWarning
             )
         return expval.real
 
+    @property
+    def _state(self):
+        """The numerical value of the current state vector.
+
+        This attribute cannot be manually overwritten.
+
+        Returns:
+            (array, tf.Tensor, torch.Tensor): the numerical tensor
+        """
+
+        return self._state_node.tensor
```

Comment:
A comment: currently, `default.qubit` and `default.gaussian` is the only device that uses `_state`; with almost everything else we use `dev.state`. @mariaschuld is currently working on an plan to standardize access to states and probabilities. Not really sure where I'm going with this, but might be worth making this `state`? Not pivotal though.

Reply:
I'm in favour of ``state``, and changing this in the default plugins as well. Irrespective of our larger strategy this is for consistency...